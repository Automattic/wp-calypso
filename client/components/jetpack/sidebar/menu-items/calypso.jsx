/**
 * External dependencies
 */
import React from 'react';

/**
 * Internal dependencies
 */
import JetpackSidebarMenuItems from '.';

export default ( { path, expandSection } ) => {
	return (
		<JetpackSidebarMenuItems
			path={ path }
			showIcons={ false }
			tracksEventNames={ {
				activityClicked: 'calypso_mysites_jetpack_sidebar_activity_clicked',
				backupClicked: 'calypso_mysites_jetpack_sidebar_backup_clicked',
				scanClicked: 'calypso_mysites_jetpack_sidebar_scan_clicked',
			} }
			expandSection={ expandSection }
		/>
	);
};
