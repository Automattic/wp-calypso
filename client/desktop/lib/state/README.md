State
==========

Provides user state functions, initially for logged in state, but easily
extended to other cases at they rise

