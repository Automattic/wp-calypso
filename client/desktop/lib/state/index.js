/**
 * Internal dependencies
 */
const keychain = require( 'desktop/lib/keychain' );

/**
 * Module variables
 */

let state = false;

function State() {
	this.loggedIn = false;
	this.user = false;
}

State.prototype.isLoggedIn = function () {
	return this.loggedIn;
};

State.prototype.login = async function ( { user, token } ) {
	this.loggedIn = true;

	if ( user && token ) {
		this.user = {
			id: user.data.ID,
			token: `${ token }`,
		};
		keychain.setUserInfo( this.user );
	}
};

State.prototype.getUser = function () {
	if ( ! this.user && this.loggedIn ) {
		this.user = keychain.getUserInfo();
	}
	return this.user;
};

State.prototype.logout = function () {
	this.loggedIn = false;
	this.user = false;
	keychain.clear();
};

State.prototype.setLogPath = function ( path ) {
	this.logPath = path;
};

State.prototype.getLogPath = function () {
	return this.logPath;
};

if ( ! state ) {
	state = new State();
}

module.exports = state;
