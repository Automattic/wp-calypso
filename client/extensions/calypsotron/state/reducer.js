/**
 * External dependencies
 */
import { combineReducers } from 'redux';

/**
 * Internal dependencies
 */
import productList from './product-list/reducer';

export default combineReducers( {
	productList,
} );
