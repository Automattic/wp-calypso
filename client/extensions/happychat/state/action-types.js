/**
 * Any new action type should be added to the set of exports below, with the
 * value mirroring its exported name.
 *
 * Please keep this list alphabetized!
 *
 * Unsure how to name an action type? Refer to our guidelines:
 *  - https://wpcalypso.wordpress.com/devdocs/docs/our-approach-to-data.md#actions
 */

export const HAPPYCHAT_BLUR = 'HAPPYCHAT_BLUR';
export const HAPPYCHAT_CONNECT = 'HAPPYCHAT_CONNECT';
export const HAPPYCHAT_CONNECTED = 'HAPPYCHAT_CONNECTED';
export const HAPPYCHAT_CONNECTING = 'HAPPYCHAT_CONNECTING';
export const HAPPYCHAT_DISCONNECTED = 'HAPPYCHAT_DISCONNECTED';
export const HAPPYCHAT_FOCUS = 'HAPPYCHAT_FOCUS';
export const HAPPYCHAT_INITIALIZE = 'HAPPYCHAT_INITIALIZE';
export const HAPPYCHAT_MINIMIZING = 'HAPPYCHAT_MINIMIZING';
export const HAPPYCHAT_OPEN = 'HAPPYCHAT_OPEN';
export const HAPPYCHAT_RECEIVE_EVENT = 'HAPPYCHAT_RECEIVE_EVENT';
export const HAPPYCHAT_RECONNECTING = 'HAPPYCHAT_RECONNECTING';
export const HAPPYCHAT_SEND_USER_INFO = 'HAPPYCHAT_SEND_USER_INFO';
export const HAPPYCHAT_SEND_MESSAGE = 'HAPPYCHAT_SEND_MESSAGE';
export const HAPPYCHAT_SET_AVAILABLE = 'HAPPYCHAT_SET_AVAILABLE';
export const HAPPYCHAT_SET_CHAT_STATUS = 'HAPPYCHAT_SET_CHAT_STATUS';
export const HAPPYCHAT_SET_GEO_LOCATION = 'HAPPYCHAT_SET_GEO_LOCATION';
export const HAPPYCHAT_SET_MESSAGE = 'HAPPYCHAT_SET_MESSAGE';
export const HAPPYCHAT_TRANSCRIPT_RECEIVE = 'HAPPYCHAT_TRANSCRIPT_RECEIVE';
export const HAPPYCHAT_TRANSCRIPT_REQUEST = 'HAPPYCHAT_TRANSCRIPT_REQUEST';
