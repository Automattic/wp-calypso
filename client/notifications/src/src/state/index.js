import { applyMiddleware, combineReducers, compose, createStore } from 'redux';

import actionMiddleware from './action-middleware';
import notes from './notes/reducer';
import suggestions from './suggestions/reducer';
import ui from './ui/reducer';

const reducer = combineReducers({
  notes,
  suggestions,
  ui,
});

/** @see https://github.com/zalmoxisus/redux-devtools-extension */
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const withMiddleware = customMiddleware =>
  composeEnhancers(applyMiddleware(actionMiddleware(customMiddleware)))(createStore);

export const init = ({ customEnhancer, customMiddleware = {} } = {}) => {
  const middle = withMiddleware(customMiddleware);
  const create = customEnhancer ? customEnhancer(middle) : middle;

  store = create(reducer, reducer(undefined, { type: '@@INIT' }));

  return store;
};

export let store = init();
