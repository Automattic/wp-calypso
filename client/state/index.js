/**
 * External dependencies
 */
import thunkMiddleware from 'redux-thunk';
import { createStore, applyMiddleware, combineReducers, compose } from 'redux';

/**
 * Internal dependencies
 */
import noticesMiddleware from './notices/middleware';
import postsEditMiddleware from './posts/middleware';
import application from './application/reducer';
import comments from './comments/reducer';
import componentsUsageStats from './components-usage-stats/reducer';
import currentUser from './current-user/reducer';
import documentHead from './document-head/reducer';
import domains from './domains/reducer';
import googleAppsUsers from './google-apps-users/reducer';
import jetpackConnect from './jetpack-connect/reducer';
import jetpackSync from './jetpack-sync/reducer';
import happychat from './happychat/reducer';
import notices from './notices/reducer';
import pageTemplates from './page-templates/reducer';
import plans from './plans/reducer';
import plugins from './plugins/reducer';
import posts from './posts/reducer';
import postTypes from './post-types/reducer';
import preferences from './preferences/reducer';
import preview from './preview/reducer';
import productsList from './products-list/reducer';
import pushNotifications from './push-notifications/reducer';
import purchases from './purchases/reducer';
import reader from './reader/reducer';
import receipts from './receipts/reducer';
import sharing from './sharing/reducer';
import signup from './signup/reducer';
import sites from './sites/reducer';
import siteSettings from './site-settings/reducer';
import stats from './stats/reducer';
import storedCards from './stored-cards/reducer';
import support from './support/reducer';
import terms from './terms/reducer';
import themes from './themes/reducer';
import ui from './ui/reducer';
import users from './users/reducer';
import wordads from './wordads/reducer';

/**
 * Module variables
 */
export const reducer = combineReducers( {
	application,
	comments,
	componentsUsageStats,
	currentUser,
	documentHead,
	domains,
	googleAppsUsers,
	happychat,
	jetpackConnect,
	jetpackSync,
	notices,
	pageTemplates,
	plugins,
	plans,
	preferences,
	preview,
	posts,
	postTypes,
	productsList,
	purchases,
	pushNotifications,
	reader,
	receipts,
	sharing,
	signup,
	sites,
	siteSettings,
	stats,
	storedCards,
	support,
	terms,
	themes,
	ui,
	users,
	wordads
} );

const middleware = [ thunkMiddleware, noticesMiddleware, postsEditMiddleware ];

if ( typeof window === 'object' ) {
	// Browser-specific middlewares
	middleware.push(
		require( './analytics/middleware.js' ).analyticsMiddleware
	);
}

let createStoreWithMiddleware = applyMiddleware.apply( null, middleware );

export function createReduxStore( initialState = {} ) {
	if (
		typeof window === 'object' &&
		window.app &&
		window.app.isDebug &&
		window.devToolsExtension
	) {
		createStoreWithMiddleware = compose( createStoreWithMiddleware, window.devToolsExtension() );
	}
	return createStoreWithMiddleware( createStore )( reducer, initialState );
}
