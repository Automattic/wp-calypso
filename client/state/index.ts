import { isEnabled } from '@automattic/calypso-config';
import { useSelector as reduxUseSelector, useDispatch as reduxUseDispatch } from 'react-redux';
import { createStore, applyMiddleware, compose, Store, StoreEnhancer } from 'redux';
import dynamicMiddlewares from 'redux-dynamic-middlewares';
import thunkMiddleware from 'redux-thunk';
import wpcomApiMiddleware from 'calypso/state/data-layer/wpcom-api-middleware';
import { addReducerEnhancer } from 'calypso/state/utils/add-reducer-enhancer';
import actionLogger from './action-log';
import consoleDispatcher from './console-dispatch';
import initialReducer from './reducer';
import { IAppState, CalypsoDispatch } from './types';

export function createReduxStore(
	initialState: Record< string, unknown >,
	reducer = initialReducer
): Store {
	const isBrowser = typeof window === 'object';
	const isDesktop = isEnabled( 'desktop' );
	const isAudioSupported = typeof window === 'object' && typeof window.Audio === 'function';

	const middlewares = [
		thunkMiddleware,
		// We need the data layer middleware to be used as early
		// as possible, before any side effects.
		// The data layer dispatches actions on network events
		// including success, failure, and progress updates
		// Its way of issuing these is to wrap the originating action
		// with special meta and dispatch it again each time.
		// If another middleware jumps in before the data layer
		// then it could mistakenly trigger on those network
		// responses. Therefore we need to inject the data layer
		// as early as possible into the middleware chain.
		wpcomApiMiddleware,
		isBrowser && require( './happychat/middleware.js' ).default,
		isBrowser && require( './happychat/middleware-calypso.js' ).default,
		dynamicMiddlewares,
		isBrowser && require( './analytics/middleware.js' ).analyticsMiddleware,
		isBrowser && require( './lib/middleware.js' ).default,
		isAudioSupported && require( './audio/middleware.js' ).default,
		isDesktop && require( './desktop/middleware.js' ).default,
	].filter( Boolean );

	const enhancers = [
		addReducerEnhancer,
		isBrowser && window.app && window.app.isDebug && consoleDispatcher,
		applyMiddleware( ...middlewares ),
		isBrowser && window.app && window.app.isDebug && actionLogger,
		isBrowser && window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
	].filter( Boolean );

	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
	// @ts-ignore The store enhancer types don't line up exactly and it's not worth fixing immediately.
	return createStore( reducer, initialState, compose< StoreEnhancer >( ...enhancers ) );
}

export const useDispatch: () => CalypsoDispatch = reduxUseDispatch;

// Basically the same types as TypedUseSelectorHook but with IAppState as the default state type.
export function useSelector< State = IAppState, Selected = unknown >(
	selector: ( state: State ) => Selected,
	equalityFn?: ( a: Selected, b: Selected ) => boolean
): Selected {
	return reduxUseSelector( selector, equalityFn );
}
