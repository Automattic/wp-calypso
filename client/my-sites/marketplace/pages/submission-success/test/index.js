/**
 * @jest-environment jsdom
 */

import { shallow } from 'enzyme';
import { Provider as ReduxProvider } from 'react-redux';
import { createReduxStore } from 'calypso/state';
import SignupSuccess from '../signup-success';

describe( 'Marketplace SignupSuccess', () => {
	const store = createReduxStore();
	const wrapper = shallow(
		<ReduxProvider store={ store }>
			<SignupSuccess />
		</ReduxProvider>
	);

	test( 'should render the page as expected', () => {
		expect( wrapper ).toMatchSnapshot();
	} );
} );
