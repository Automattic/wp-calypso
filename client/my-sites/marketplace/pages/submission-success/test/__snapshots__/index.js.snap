// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marketplace SignupSuccess should render the page as expected 1`] = `
<ThemeProvider
  theme={
    Object {
      "fontSize": Object {
        "small": "14px",
      },
      "weights": Object {
        "bold": "600",
        "normal": "400",
      },
    }
  }
>
  <EmotionGlobal
    styles={
      Object {
        "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInNpZ251cC1zdWNjZXNzLnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiQUFnQmdCIiwiZmlsZSI6InNpZ251cC1zdWNjZXNzLnRzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCB7IEdyaWRpY29uIH0gZnJvbSAnQGF1dG9tYXR0aWMvY29tcG9uZW50cyc7XG5pbXBvcnQgeyBUaGVtZVByb3ZpZGVyLCBHbG9iYWwsIGNzcyB9IGZyb20gJ0BlbW90aW9uL3JlYWN0JztcbmltcG9ydCB7IEJ1dHRvbiB9IGZyb20gJ0B3b3JkcHJlc3MvY29tcG9uZW50cyc7XG5pbXBvcnQgeyB1c2VUcmFuc2xhdGUgfSBmcm9tICdpMThuLWNhbHlwc28nO1xuaW1wb3J0IHN1Ym1pc3Npb25TdWNjZXNzSW1hZ2UgZnJvbSAnY2FseXBzby9hc3NldHMvaW1hZ2VzL21hcmtldHBsYWNlL3N1Ym1pc3Npb24tc3VjY2Vzcy5wbmcnO1xuaW1wb3J0IHRoZW1lIGZyb20gJ2NhbHlwc28vbXktc2l0ZXMvbWFya2V0cGxhY2UvdGhlbWUnO1xuaW1wb3J0IE1hc3RlcmJhclN0eWxlZCBmcm9tICcuLi8uLi9jb21wb25lbnRzL21hc3RlcmJhci1zdHlsZWQnO1xuaW1wb3J0ICcuL3N0eWxlLnNjc3MnO1xuXG5jb25zdCBTaWdudXBTdWNjZXNzID0gKCkgPT4ge1xuXHRjb25zdCB0cmFuc2xhdGUgPSB1c2VUcmFuc2xhdGUoKTtcblxuXHRyZXR1cm4gKFxuXHRcdDxUaGVtZVByb3ZpZGVyIHRoZW1lPXsgdGhlbWUgfT5cblx0XHRcdHsgLyogVXNpbmcgR2xvYmFsIHRvIG92ZXJyaWRlIEdsb2JhbCBtYXN0ZXJiYXIgaGVpZ2h0ICovIH1cblx0XHRcdDxHbG9iYWxcblx0XHRcdFx0c3R5bGVzPXsgY3NzYFxuXHRcdFx0XHRcdGJvZHkuaXMtc2VjdGlvbi1tYXJrZXRwbGFjZSB7XG5cdFx0XHRcdFx0XHQtLW1hc3RlcmJhci1oZWlnaHQ6IDcycHg7XG5cdFx0XHRcdFx0fVxuXHRcdFx0XHRgIH1cblx0XHRcdC8+XG5cdFx0XHQ8TWFzdGVyYmFyU3R5bGVkXG5cdFx0XHRcdG9uQ2xpY2s9eyAoKSA9PiAoIGRvY3VtZW50LmxvY2F0aW9uLmhyZWYgPSBgJHsgZG9jdW1lbnQubG9jYXRpb24ub3JpZ2luIH1gICkgfVxuXHRcdFx0XHRiYWNrVGV4dD17IHRyYW5zbGF0ZSggJ0JhY2sgdG8gV29yZFByZXNzLmNvbScgKSB9XG5cdFx0XHQvPlxuXHRcdFx0PGRpdiBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc1wiPlxuXHRcdFx0XHQ8ZGl2IGNsYXNzTmFtZT1cInNpZ251cC1zdWNjZXNzX19oZWFkZXJcIj5cblx0XHRcdFx0XHQ8aW1nIHNyYz17IHN1Ym1pc3Npb25TdWNjZXNzSW1hZ2UgfSBhbHQ9XCJTdWJtaXNzaW9uIHN1Y2Nlc3NcIiAvPlxuXHRcdFx0XHRcdDxoMSBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc19faGVhZGVyLXRpdGxlIHdwLWJyYW5kLWZvbnRcIj5cblx0XHRcdFx0XHRcdHsgdHJhbnNsYXRlKCAnV2XigJlsbCBiZSBpbiB0b3VjaCcgKSB9XG5cdFx0XHRcdFx0PC9oMT5cblx0XHRcdFx0XHQ8cD5cblx0XHRcdFx0XHRcdHsgdHJhbnNsYXRlKFxuXHRcdFx0XHRcdFx0XHQnVGhhbmsgeW91IGZvciBhcHBseSB0byBqb2luIHRoZSBXb3JkUHJlc3MuY29tIG1hcmtldHBsYWNlLiBXZeKAmWxsIGJlIGluIHRvdWNoIHdpdGggeW91IHZlcnkgc29vbiB0byBkaXNjdXNzIG5leHQgc3RlcHMuJ1xuXHRcdFx0XHRcdFx0KSB9XG5cdFx0XHRcdFx0PC9wPlxuXHRcdFx0XHQ8L2Rpdj5cblx0XHRcdFx0PGRpdiBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc19fYm9keVwiPlxuXHRcdFx0XHRcdDxkaXYgY2xhc3NOYW1lPVwic2lnbnVwLXN1Y2Nlc3NfX3Jvd1wiPlxuXHRcdFx0XHRcdFx0PGRpdiBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc19fcm93LWljb25cIj5cblx0XHRcdFx0XHRcdFx0PEdyaWRpY29uIGljb249XCJjYXJ0XCIgLz5cblx0XHRcdFx0XHRcdDwvZGl2PlxuXHRcdFx0XHRcdFx0PGRpdiBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc19fcm93LWNvbnRlbnRcIj5cblx0XHRcdFx0XHRcdFx0PGgyPnsgdHJhbnNsYXRlKCAnTGVhcm4gTW9yZScgKSB9PC9oMj5cblx0XHRcdFx0XHRcdFx0PHA+eyB0cmFuc2xhdGUoICdSZWFkIG1vcmUgYWJvdXQgc2VsbGluZyBvbiB0aGUgV29yZFByZXNzLmNvbSBtYXJrZXRwbGFjZS4nICkgfTwvcD5cblx0XHRcdFx0XHRcdDwvZGl2PlxuXHRcdFx0XHRcdFx0PEJ1dHRvbiBpc1NlY29uZGFyeSBocmVmPVwiaHR0cHM6Ly9kZXZlbG9wZXIud29yZHByZXNzLmNvbS93b3JkcHJlc3MtY29tLW1hcmtldHBsYWNlL1wiPlxuXHRcdFx0XHRcdFx0XHR7IHRyYW5zbGF0ZSggJ0NvbnRpbnVlJyApIH1cblx0XHRcdFx0XHRcdDwvQnV0dG9uPlxuXHRcdFx0XHRcdDwvZGl2PlxuXHRcdFx0XHRcdDxociAvPlxuXHRcdFx0XHRcdDxkaXYgY2xhc3NOYW1lPVwic2lnbnVwLXN1Y2Nlc3NfX3Jvd1wiPlxuXHRcdFx0XHRcdFx0PGRpdiBjbGFzc05hbWU9XCJzaWdudXAtc3VjY2Vzc19fcm93LWljb25cIj5cblx0XHRcdFx0XHRcdFx0PEdyaWRpY29uIGljb249XCJjb2dcIiAvPlxuXHRcdFx0XHRcdFx0PC9kaXY+XG5cdFx0XHRcdFx0XHQ8ZGl2IGNsYXNzTmFtZT1cInNpZ251cC1zdWNjZXNzX19yb3ctY29udGVudFwiPlxuXHRcdFx0XHRcdFx0XHQ8aDI+eyB0cmFuc2xhdGUoICdWaWV3IHRoZSBtYXJrZXRwbGFjZScgKSB9PC9oMj5cblx0XHRcdFx0XHRcdFx0PHA+eyB0cmFuc2xhdGUoICdTaWduIGluIHRvIHNlZSB3aGF0IHRoZSBtYXJrZXRwbGFjZSBoYXMgdG8gb2ZmZXIuJyApIH08L3A+XG5cdFx0XHRcdFx0XHQ8L2Rpdj5cblx0XHRcdFx0XHRcdDxCdXR0b24gaXNTZWNvbmRhcnkgaHJlZj1cIi9wbHVnaW5zXCI+XG5cdFx0XHRcdFx0XHRcdHsgdHJhbnNsYXRlKCAnQ29udGludWUnICkgfVxuXHRcdFx0XHRcdFx0PC9CdXR0b24+XG5cdFx0XHRcdFx0PC9kaXY+XG5cdFx0XHRcdDwvZGl2PlxuXHRcdFx0PC9kaXY+XG5cdFx0PC9UaGVtZVByb3ZpZGVyPlxuXHQpO1xufTtcblxuZXhwb3J0IGRlZmF1bHQgU2lnbnVwU3VjY2VzcztcbiJdfQ== */",
        "name": "sizr66-SignupSuccess",
        "styles": "body.is-section-marketplace{--masterbar-height:72px;};label:SignupSuccess;",
        "toString": [Function],
      }
    }
  />
  <MasterbarStyled
    backText="Back to WordPress.com"
    onClick={[Function]}
  />
  <div
    className="signup-success"
  >
    <div
      className="signup-success__header"
    >
      <img
        alt="Submission success"
        src="submission-success.png"
      />
      <h1
        className="signup-success__header-title wp-brand-font"
      >
        We’ll be in touch
      </h1>
      <p>
        Thank you for apply to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.
      </p>
    </div>
    <div
      className="signup-success__body"
    >
      <div
        className="signup-success__row"
      >
        <div
          className="signup-success__row-icon"
        >
          <Gridicon
            icon="cart"
          />
        </div>
        <div
          className="signup-success__row-content"
        >
          <h2>
            Learn More
          </h2>
          <p>
            Read more about selling on the WordPress.com marketplace.
          </p>
        </div>
        <ForwardRef(Button)
          href="https://developer.wordpress.com/wordpress-com-marketplace/"
          isSecondary={true}
        >
          Continue
        </ForwardRef(Button)>
      </div>
      <hr />
      <div
        className="signup-success__row"
      >
        <div
          className="signup-success__row-icon"
        >
          <Gridicon
            icon="cog"
          />
        </div>
        <div
          className="signup-success__row-content"
        >
          <h2>
            View the marketplace
          </h2>
          <p>
            Sign in to see what the marketplace has to offer.
          </p>
        </div>
        <ForwardRef(Button)
          href="/plugins"
          isSecondary={true}
        >
          Continue
        </ForwardRef(Button)>
      </div>
    </div>
  </div>
</ThemeProvider>
`;
